testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -1291,3 +1292,20 @@ def test_empty_creation_raises_error(self):
         with pytest.raises(TypeError) as exc:
             u = mda.Universe()
         assert 'Universe.empty' in str(exc.value)
+
+
+def test_as_Universe_deprecation():
+    with pytest.deprecated_call():
+        _ = mda.core.universe.as_Universe(PSF, DCD)
+
+
+def test_deprecate_b_tempfactors():
+    u = mda.Universe(PDB)
+    with pytest.warns(DeprecationWarning, match="alias"):
+        u.add_TopologyAttr("bfactors")
+
+
+def test_deprecate_temp_bfactors():
+    u = mda.Universe(MMTF)
+    with pytest.warns(DeprecationWarning, match="alias"):
+        u.add_TopologyAttr("tempfactors")
```

Comment:
@lilyminium is this still necessary?

Reply:
for alias bfactors == tempfactors? I guess I'd say leave the warning in until we actually alias them